src/content/blog/2023_mitiq_stim_workflow.md

```diff
@@ -0,0 +1,41 @@
+---
+title: ZNE on logical circuits with Mitiq and the Stim backend 
+author: Misty Wahl
+day: 14
+month: 9
+year: 2023
+---
+
+
+In Mitiq v0.29.0 we added a [tutorial demonstrating a method of combining quantum error mitigation (QEM) and quantum error correction (QEC)](https://mitiq.readthedocs.io/en/stable/examples/zne_logical_rb_cirq_stim.html), with the goal of reducing the effective logical error rate of the computation.
+While QEM techniques such as zero noise extrapolation (ZNE) and probabilistic error cancellation are typically thought of as belonging to the NISQ regime, recently it has been shown that they can also benefit applications within the fault-tolerant regime [^1][^2][^3].
+In this example ZNE is applied with noise scaling by global unitary folding on logical randomized benchmarking (RB) circuits, and the use of Mitiq with the Stim stabilizer simulator as the backend is introduced.
+
+## About Mitiq
+[Mitiq](https://mitiq.readthedocs.io/en/latest/) is a compiler that makes programs more robust to errors in quantum computers.
+Because quantum computers have high error rates, compiling for error robustness is critical for useful applications.
+Mitiq is the leading quantum error mitigation compiler and has recently hit 100k downloads and is developed by a community of over 50 contributors worldwide.
+
+## About Stim
+Stim is a quantum stabilizer circuit simulation package, specialized for the simulation of quantum error correction (QEC) circuits.
+It can be used as a Python package, a command line tool, or a C++ library.
+More information can be found on the [Stim project page](https://github.com/quantumlib/Stim). 
+
+Although Mitiq does not support Stim directly as a frontend, it can interface with the Stim backend through the Cirq and Stimcirq packages. Stimcirq is a library for translating between Cirq and Stim.
+
+## A Mitiq + Stim tutorial for applying error mitigation on logical circuits
```

Comment:
The text paragraph is a bit long here. What about breaking it into two, with two titles, one focusing on the content and one speaking about open science and the genesis of it? Just a proposal.

Reply:
Split it into two sections, "A Mitiq + Stim tutorial for applying error mitigation on logical circuits" and "Quantum community impact"